main.py

```diff
@@ -65,15 +65,6 @@ def notify():
         logger.info(message)
         requests.post(slack_webhook_url, json={'text': message})
 
-
-# TODO:
-# We should create a package and move this into that package
-@app.route('/photo', methods=['GET'])
```

Comment:
This mean after this patch, `/photo` is no longer working.
Can we also add an `apis/photo.py` to maintain compatibility?

Reply:
OK, I will add it.

Reply:
971363cdb4a19f5663f18de5731291081d5b38cf